### ISSUE TITLE
request for seekworld_test.parquet

### ISSUE THREAD

**Issue by zy5037** [Visual-Agent/DeepEyes#15]
Hello, neither Hugging Face nor GitHub provides the seekworld_test.parquet validation set.

**Comment by ChenShawn** [comment]
Sorry for the mistake.
As you can see in [the training script](https://github.com/Visual-Agent/DeepEyes/blob/main/examples/agent/final_merged_v1v8_thinklite.sh#L65), `trainer.test_freq` is set to 10000, which means there will be no validation during training.
Therefore, the file `seekworld_test.parquet` is just a random placeholder that is never used in training or validation. It is not relevant to any of the experiments in our paper.

We have updated our code to remove the dependency for this file from the script.
